src/main/java/uk/gov/hmcts/reform/sscs/service/AbstractFooterService.java

```diff
@@ -39,7 +39,12 @@ public FooterDetails addFooterToExistingToContentAndCreateNewUrl(DocumentLink ur
 
             String bundleAddition = getNextBundleAddition(documents);
 
-            String bundleFileName = overrideFileName != null ? overrideFileName : buildBundleAdditionFileName(bundleAddition, label + " issued on " + dateIssued);
+            String verb = " issued on ";
+            if (DocumentType.SET_ASIDE_APPLICATION.equals(documentType)) {
```

Comment:
easly an ternary operator could be used here

Reply:
done